pkg/bosh/manifest/kube_converter.go

```diff
@@ -852,3 +819,39 @@ func (m *Manifest) ApplyBPMInfo(kubeConfig *KubeConfig, allResolvedProperties ma
 	}
 	return nil
 }
+
+// AppendToContainerList will append one container to a v1.Container{}
+func (m *Manifest) AppendToContainerList(job Job, releaseImage string, containerReleasePath string, volumeMountName string) []v1.Container {
```

Comment:
This is not appending to anything, you're just returning an array with one `v1.Container`.
The function should be: 
`jobSpecCopierContainer(containerName, releaseImage, volumeMountName, containerMountPath string) v1.Container`

`containerReleasePath` shouldn't be variable - let's always make it `/var/vcap/all-releases/jobs-src`.
Then we need to change the data gatherer and renderer to use that by default.

Reply:
I just addressed your requested changes.